timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -367,6 +367,7 @@ def fit(
         refit_every_n_windows: int = 1,
         refit_full: bool = False,
         enable_ensemble: bool = True,
+        split_time_limit: Optional[bool] = None,
```

Comment:
I don't follow why this is needed. This is a complication Tabular did not need to have. What's unique to time series that requires this? Can't we default to what Tabular is doing? This feels like a choice to be made in favor of cross-AG consistency at least as much as some of the others we've looked at.

Moreover, I want to force us to think well about the API here. Is this the only way in which users can wish to play with the time limit? Or is this a `time_limit_policy="split"` that can take different values in the future?

Also, in the comments we point out that the best quality search space is a dummy. Is it? I think several models are still trained for DeepAR so hyperparameter search is performed.

Reply:
I agree with you, the API becomes really confusing with this change. I have updated the PR to always split the time limit and remove the option to control it. I will do benchmarking before merging this PR to see whether this change is beneficial.